alert/alert.css

```diff
@@ -0,0 +1,73 @@
+:root {
+  --primary-color: #36a5ff;
```

Comment:
Since these colors might be common to other stylesheets, suggest adding the variables to a common sheet. Say variables.css

Reply:
yes will do that, thank you for review